crates/bindings-csharp/Runtime/LogStopwatch.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SpacetimeDB
+{
+    public sealed class LogStopwatch : IDisposable
+    {
+        private uint StopwatchId;
+
+        public LogStopwatch(string name)
+        {
+            StopwatchId = StartStopwatchInternal(name);
+        }
+
+        private static uint StartStopwatchInternal(string name)
+        {
+            var name_bytes = Encoding.UTF8.GetBytes(name);
+            var id = Internal.FFI._console_timer_start(name_bytes, (uint)name_bytes.Length);
+            return id;
+        }
+
+        void IDisposable.Dispose()
+        {
+            End();
```

Comment:
I believe we'll have a problem if someone does `using var ...` but decides to `.End()` manually early somewhere. In Rust it's not an issue because `end` consumes the instance by value, so `Drop` is guaranteed to run only once, but in C# that's not the case so we'll have a risk of double-free, leading to an error due to the timer already having been deleted.

Reply:
Yeah that's pretty reasonable.